### PR TITLE
Improve SvgPicture size handling

### PR DESCRIPTION
**Pull request by sroddy**
See #57

### CODE REVIEW

**Comment by dnfield** [comment]
I think this looks good.  Can you merge in latest changes from master and see if that fixes up CI?

**Comment by sroddy** [comment]
@dnfield I've rebased and pushed a missing thing into one of the constructors. The CI is now happy.
It may be worth to review a bit the docs to reflect the new behaviour, but I'm not really good at it. Can you please have a look? Thank you

File: lib/svg.dart [hunk 1]
```diff
@@ -450,6 +462,10 @@ class SvgPicture extends StatefulWidget {
   /// will take the height of its parent.
   final double height;
 
+  final BoxFit fit;
```
Comment (dnfield) [inline lib/svg.dart]: /// How to inscribe the picture into the space allocated during layout.  The default is [BoxFit.contain].

File: lib/svg.dart [hunk 2]
```diff
@@ -450,6 +462,10 @@ class SvgPicture extends StatefulWidget {
...
+
+  final Alignment alignment;
```
Comment (dnfield) [inline lib/svg.dart]:
```
  /// How to align the picture within its parent widget.
  ///
  /// The alignment aligns the given position in the picture to the given position
  /// in the layout bounds. For example, an [Alignment] alignment of (-1.0,
  /// -1.0) aligns the image to the top-left corner of its layout bounds, while a
  /// [Alignment] alignment of (1.0, 1.0) aligns the bottom right of the
  /// picture with the bottom right corner of its layout bounds. Similarly, an
  /// alignment of (0.0, 1.0) aligns the bottom middle of the image with the
  /// middle of the bottom edge of its layout bounds.
  ///
  /// If the [alignment] is [TextDirection]-dependent (i.e. if it is a
  /// [AlignmentDirectional]), then a [TextDirection] must be available
  /// when the picture is painted.
  ///
  /// Defaults to [Alignment.center].
  ///
  /// See also:
  ///
  ///  * [Alignment], a class with convenient constants typically used to
  ///    specify an [AlignmentGeometry].
  ///  * [AlignmentDirectional], like [Alignment] for specifying alignments
  ///    relative to text direction.
```

**Comment by dnfield** [comment]
Added some doc suggestions, feel free to tweak.  They're mainly adapted from similar places in the Flutter framework.

**Pull request merged**